test-lib/test.js

```diff
@@ -1,15 +1,25 @@
 const fs = require('fs-extra');
 const path = require('path');
+const { stripIndent } = require('common-tags');
 
-if (!fs.existsSync(path.join(__dirname, '/../test/node_modules'))) {
-  fs.mkdirSync(path.join(__dirname, '/../test/node_modules'));
-  fs.symlinkSync(path.join(__dirname, '/..'), path.join(__dirname, '/../test/node_modules/apostrophe'), 'dir');
-}
+fs.removeSync(path.join(__dirname, '/../test/node_modules'));
+fs.mkdirSync(path.join(__dirname, '/../test/node_modules'));
+fs.symlinkSync(path.join(__dirname, '/..'), path.join(__dirname, '/../test/node_modules/apostrophe'), 'dir');
 
-if (!fs.existsSync(path.join(__dirname, '/../test/package.json'))) {
-  // Linking package.json file for the start up build task to find.
-  fs.ensureSymlink(path.join(__dirname, '/../package.json'), path.join(__dirname, '/../test/package.json'));
+// Need a "project level" package.json for functionality that checks
+// whether packages in node_modules are project level or not
 
-}
+const packageJson = path.join(__dirname, '/../test/package.json');
+// Remove it first, in case it's the old-style symlink to the main package.json,
+// which would break
+fs.removeSync(packageJson);
+fs.writeFileSync(packageJson, stripIndent`
```

Comment:
I don't think `stripIndent` is helping here since the block isn't indented.

Reply:
Cleans up the leading newline, so we can open it normally on this line and not the next which would throw off the visual balancing of the curlies. May as well use it.